docker-compose.yml

```diff
@@ -114,6 +114,7 @@ services:
       dockerfile: ./docker/orthanc-anon/Dockerfile
       args:
         <<: *build-args-common
+        PROJECT_CONFIGS_DIR: ${PROJECT_CONFIGS_DIR}
```

Comment:
Probably better to just mount it as read only instead of copying it into the image